Make function get_progress be compatible with more video format (#3381)
openvinotoolkit/cvat: CHANGELOG.md

- Duplication of the cuboids when redraw them (<https://github.com/openvinotoolkit/cvat/pull/3308>)
- Some code issues in Deep Extreme Cut handler code (<https://github.com/openvinotoolkit/cvat/pull/3325>)
- UI fails when inactive user is assigneed to a task/job (<https://github.com/openvinotoolkit/cvat/pull/3343>)

> - Calculate precise progress of decoding a video file (https://github.com/openvinotoolkit/cvat/pull/3381)

Reviewer: ```suggestion
- Calculate precise progress of decoding a video file (<https://github.com/openvinotoolkit/cvat/pull/3381>)
```

> - Can't calculat precise progress of image extraction and compression when operating video with particular format (https://github.com/openvinotoolkit/cvat/pull/3381)

Reviewer: ```suggestion
- Calculate precise progress of decoding a video file (https://github.com/openvinotoolkit/cvat/pull/3381)
```
